### ISSUE TITLE
Scope `/api/stream` to a specific user

### ISSUE THREAD

**Issue by wohlner** [livepeer/studio#256]
Return all streams by a user. Should do the same thing that `/api/stream/user/<userId>` currently does.

The `200` response for the new endpoint and `/api/stream/user/<userId>` should be the same

This may be relevant https://github.com/livepeer/livepeer-com/blob/master/packages/api/src/schema/schema.yaml

**Comment by darkdarkdragon** [comment]
@wohlner what is the reason for this change?

**Comment by wohlner** [comment]
@darkdarkdragon I want API users to get a list of their streams via` /api/stream` so that they can more easily sort the list of streams by endpoint parameter. Right now, we have to create custom filters to sort the `/api/stream/user/` response. And that's not intuitive.

**Comment by wohlner** [comment]
@julianbenegas This is now deployed on stage to livepeer.monster. I just tested a GET request to https://livepeer.monster/api/stream using a token for a non-admin .monster account and got this error {"errors":["user can only request information on their own streams"]}.

And then when I made the same GET request using an admin .monster token, the request response is super slow/doesn't work. I got a 524 error, the response timed out.

Both these GET requests to /api/stream should return a list of all the API key holder's stream objects.

cc:@darkdragon#2215
